<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1381</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>550</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>541</width>
      <height>131</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="MainTab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QPushButton" name="buttonConfig">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>CONFIG</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonOnce">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>ONCE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonStop">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonStart">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ManualTab">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>531</width>
        <height>91</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>MoveTo
PickUP
XY Position</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>MoveTo
Place
XY Position</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>MoveTo
OffSet
XY Position</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>PickUp
Motion</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Place
Motion</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="IOManagerTab">
     <attribute name="title">
      <string>IOManager</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonMotors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>99</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Motors</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonIOs">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>99</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>IOs</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBoxProcessImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Processed Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_data">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>490</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="TrackingLabel" name="labelTrackingMouse">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSummaryOnScreen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>20</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TrackingLabel</class>
   <extends>QLabel</extends>
   <header location="global">trackinglabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
